src/main/java/edu/rosehulman/quota/javasdk/QuotaClient.java

```diff
@@ -32,7 +32,7 @@ static synchronized QuotaClient getInstance() {
   protected boolean addUser(Partner partner, Product product, String userId) {
     try {
       // TODO: Put Quota Server path in config and in here
-      HttpResponse<String> response = Unirest.post("http://quota.csse.rose-hulman.edu:8080/partner/{partnerId}/product/{productId}/user/{userId}").routeParam("partnerId", partner.getPartnerId()).routeParam("productId", product.getProductId()).routeParam("userId", userId).asString();
+      HttpResponse<String> response = Unirest.post("http://quota.csse.rose-hulman.edu:8080/partner/{partnerId}/product/{productId}/user/{userId}").routeParam("partnerId", partner.getPartnerId()).routeParam("productId", product.getProductId()).body("{\"id\":\"" + userId + "\"}").asString();
```

Comment:
Doesn't {userId} need to be removed from the end of this path since it was removed in Quota?

Reply:
It's in the body now